import/handle_data.py

```
@@ -0,0 +1,77 @@
+# This module will get data from Google Sheets and format it as a list of dicts
+import time
+import difflib
+
+# Lowercase email addresses and remove whitespace.
+def normalize_email(email):
+    return email.lower().replace(" ", "")
+
+def create_display_name(first_name, last_name):
+    return first_name + ' ' + last_name
+
+# Convert a Google Sheet timestamp string ('5/1/2018 21:07:52') to basic date string format ('2018-5-1')
+def timestamp_to_date(ts):
+    dt_tup = time.strptime(ts, "%m/%d/%Y %H:%M:%S")
+    return str(dt_tup[0]) + "-" + str(dt_tup[1]) + "-" + str(dt_tup[2])
+
+# Clean up the building/street/unit number fields so we have a cohesive address.
+def format_address(building, street, unit):
+    building_street = building + ' ' + street
+    return building_street if unit == '' else building_street + ', ' + unit
+
+# Convert banked hours to a float.
+def convert_hours(hours):
+    if hours == '#NUM!' or hours == '':
+        return float(0)
+    else:
+        return float(hours)
+
+# Convert the string from the new owner sheet into a category matching the types in the owner_type table.
+def convert_owner_type(owner_type):
```

Perhaps we just want to have a simple dict to do the mapping? This is nice since it's automatic but it seems like it might fail silently/behave unexpectedly without us knowing unless we review the output closely.